src/Lumper.Lib/BSP/Lumps/GameLumps/StaticPropLump.cs

```diff
@@ -168,7 +139,14 @@ protected override void ReadItem(BinaryReader reader)
             prop.FlagsEx = reader.ReadUInt32();
         // since v11
         if (ActualVersion >= StaticPropVersion.V11)
-            prop.UniformScale = System.BitConverter.ToSingle(reader.ReadBytes(4));
+        {
+            var x = reader.ReadSingle();
+            prop.UniformScale = ActualVersion < StaticPropVersion.V13
+                ? new Vector3 { X = x, Y = x, Z = x }
+                : new Vector3 { X = x, Y = reader.ReadSingle(), Z = reader.ReadSingle() };
+        }
+
+
```

Comment:
part of staticprop v13 and should be in a commit. Your commt message says you had problems splitting, but this shouldn't be so hard + there are v13 changes missing from this commit I think

Reply:
Ugh, lost the original commit since I deleted your branch, sorry. I found one commit in reflog with the original commit but I don't think it's the same as your original commit.

This took me about 45 minutes to do, I could probably spend longer and do a better job, but I just don't think it's a good use of my time. If something seems weird, I suggest looking at this file in the "All files changed" and making sure it makes sense there. I'm about to get through and fixup some other commits to make your review easier, but splitting out two commits just for the sake of the commit history being cleaner is too time-consuming and tedious.

Reply:
I can push my branch again if you want

Reply:
If you really want, go ahead.

Reply:
Okay well I don't have the commit and don't have time to rework these commits, don't want to delay this PR further just for sake of Git history